doc/release-notes.md

```diff
@@ -4,3 +4,50 @@ release-notes at release time)
 Notable changes
 ===============
 
+Sprout to Sapling Migration Tool
+--------------------------------
+This release includes the addition of a tool that will enable users to migrate
+shielded funds from the Sprout pool to the Sapling pool while minimizing
+information leakage. 
+
+The migration can be enabled using the RPC `z_setmigration` or by including
+`-migration` in the `zcash.conf` file. Unless otherwise specified funds will be
+migrated to the wallet's default Sapling address; it is also possible to set the 
+receiving Sapling address using the `-migrationdestaddress` option in `zcash.conf`.
```

Comment:
This may be documentation convention, in which case ignore this comment, but the lines added to `zcash.conf` don't start with a dash. I was hoping https://github.com/zcash/zips/blob/master/zip-0308.rst would explain exactly how to enable migration in `zcash.conf` but it does not. Do you think this should be stated explicitly either here or there:
```
migration=1
```
(and perhaps similarly for `migrationdestaddress`). Again, if this is conventional knowledge, then that's fine.

Reply:
I changed this and pushed over the original commit.